task_test.go

```diff
@@ -1086,12 +1089,23 @@ func TestIncludesRemote(t *testing.T) {
 
 	dir := "testdata/includes_remote"
 
-	srv := httptest.NewServer(http.FileServer(http.Dir(dir)))
+	cwd, _ := os.Getwd()
+	gitHandler, gitSupported := createGitHTTPHandler(t, cwd)
+	if !gitSupported {
+		t.Log("git tests not supported and will be skipped")
+	}
```

Comment:
I put this as a guard for now, but I would expect pretty much every CI and dev environment will have `git` installed.  If you all agree, I will remove this guard to reduce noise in the code.